### PR TITLE
php update

### PR DESCRIPTION
**Pull request by kellymore**

### CODE REVIEW

File: pages/guides/webhooks/validating-webhooks.mdx [hunk 1]
```diff
@@ -62,37 +62,46 @@ not available
...
-  return True if (signature == proposed_signature) else False
+  return hmac.compare_digest(signature, proposed_signature)
 ```
 
 ```php
 <?php
-function verifyGatewaySignature($proposedSignature, $webhookSecret, $payloadBody) {
-    $signature = hash_hmac("sha256", $payloadBody, $webhookSecret);
-    return $signature == $proposedSignature;
+function verifyGatewaySignature($proposedSignature, $webhookSecret,
```
Comment (sausman) [inline pages/guides/webhooks/validating-webhooks.mdx]: The formatting looks a little off on this one.

File: pages/guides/webhooks/validating-webhooks.mdx [hunk 2]
```diff
@@ -62,37 +62,46 @@ not available
 ```
 
 ```javascript
-var verifyGatewaySignature = function (
+var crypto = require("crypto");
+const fromBuffer = require("buffer").Buffer.from;
```
Comment (sausman) [inline pages/guides/webhooks/validating-webhooks.mdx]: Maybe we make this `var` instead of `const` (for now) just to be consistent?

Comment (sausman) [reply pages/guides/webhooks/validating-webhooks.mdx]: Also, let's rename this to `bufferFrom` (my bad)

Comment (kellymore) [reply pages/guides/webhooks/validating-webhooks.mdx]: ok

File: pages/guides/webhooks/validating-webhooks.mdx (hunk 1 above)
Comment (kellymore) [reply pages/guides/webhooks/validating-webhooks.mdx]: will fix asap

**Comment by kellymore** [comment]
@sausman ok fixed everything. Good to squash and merge?

**Pull request merged**